plugins/cron/src/main/java/org/crsh/cron/CronPlugin.java

```diff
@@ -182,9 +182,10 @@ public TaskTable getTasks() {
     //
     Resource res = getConfig();
     List<String> lines = null;
+    BufferedReader reader = null;
     try {
       lines = new ArrayList<String>();
-      BufferedReader reader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(res.getContent()), "UTF-8"));
+      reader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(res.getContent()), "UTF-8"));

```

Comment:
Instead, use try-with-resources for cleaner code, and don't force a particular encoding; see my comments in your pull request about character sets.